### PR TITLE
Bugherd # 698 - Removing reference to Esper

### PR DESCRIPTION
**Pull request by MWindel**
Removed sentence in Cockpit > Smart rules which was refering to Esper.

All changes will be cherry-picked to the respective release branches.

### CODE REVIEW

**Review by BeateRixen — APPROVED** [review]

File: content/users-guide/cockpit-bundle/smart-rules.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ helpcontent:
 - [Administration > Managing business rules](/users-guide/administration#business-rules) for details on managing smart rules for your devices.
 {{< /c8y-admon-related >}}
 
-{{< product-c8y-iot >}} includes a rule engine to analyze data in realtime and to perform actions based on data. These rules are specified in a scripting language and are managed in the [Administration application](/users-guide/administration).
+{{< product-c8y-iot >}} includes a rule engine to analyze data in realtime and to perform actions based on data.
```
Comment (rthrippleton) [inline content/users-guide/cockpit-bundle/smart-rules.md:18]: Realising that "rule engine" isn't a good description for the streaming analytics component:
```suggestion
{{< product-c8y-iot >}} includes Streaming Analytics which can analyze data in realtime and perform actions based on data.
```

Comment (rthrippleton) [reply content/users-guide/cockpit-bundle/smart-rules.md:18]: You might also want to xref to the Streaming Analytics guide.

**Review by rthrippleton — CHANGES_REQUESTED** [review]

File: content/users-guide/cockpit-bundle/smart-rules.md (hunk 1 above)
Comment (kailol-sag) [reply content/users-guide/cockpit-bundle/smart-rules.md:18]: I'd put it as "... includes the Streaming Analytics application which ...."

**Comment by MWindel** [comment]
@rthrippleton @kailol-sag In which target versions should this be as well?

File: content/users-guide/cockpit-bundle/smart-rules.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ helpcontent:
 - [Administration > Managing business rules](/users-guide/administration#business-rules) for details on managing smart rules for your devices.
 {{< /c8y-admon-related >}}
 
-{{< product-c8y-iot >}} includes a rule engine to analyze data in realtime and to perform actions based on data. These rules are specified in a scripting language and are managed in the [Administration application](/users-guide/administration).
+{{< product-c8y-iot >}} includes Streaming Analytics which can analyze data in realtime and perform actions based on data.
```
Comment (kailol-sag) [inline content/users-guide/cockpit-bundle/smart-rules.md:18]:
```suggestion
{{< product-c8y-iot >}} includes the Streaming Analytics application which can analyze data in realtime and perform actions based on data.
```

**Pull request merged**